### PR TITLE
CI: Run clang-tidy

### PR DESCRIPTION
**Pull request by fancycode**
This can help finding some errors / possible optimizations.

### CODE REVIEW

**Review by farindk — COMMENTED** [review]
Fine.

**Comment by farindk** [comment]
Thank you, Jojo.

File: libheif/heif_colorconversion.cc [hunk 1]
```diff
@@ -856,10 +856,10 @@ Op_YCbCr420_to_RGB24::convert_colorspace(const std::shared_ptr<const HeifPixelIm
...
   }
 
-  int r_cr = static_cast<int>(lround(256 * coeffs.r_cr));
-  int g_cr = static_cast<int>(lround(256 * coeffs.g_cr));
-  int g_cb = static_cast<int>(lround(256 * coeffs.g_cb));
-  int b_cb = static_cast<int>(lround(256 * coeffs.b_cb));
+  int r_cr = static_cast<int>(std::lround(256 * coeffs.r_cr));
+  int g_cr = static_cast<int>(std::lround(256 * coeffs.g_cr));
+  int g_cb = static_cast<int>(std::lround(256 * coeffs.g_cb));
+  int b_cb = static_cast<int>(std::lround(256 * coeffs.b_cb));
```
[the remark this answers is not in the document]
Comment (fancycode) [reply libheif/heif_colorconversion.cc:862]: Yeah, although this invocation here is only once per conversion, not once per pixel as far as I understand. So not *that* time critical.

Anyway, I would rather change the `lround`  to a simple cast in a different PR.

[the remark this answers is not in the document]
Comment (farindk) [reply libheif/heif_colorconversion.cc:862]: Right, I just saw`lround` in the tiny diff at github and red warning lights turned on in my head. Yes, we can keep them.

**Pull request merged**